state/state.go

```diff
@@ -8,11 +8,13 @@ import (
 	"time"
 
 	abci "github.com/tendermint/abci/types"
+
```

Comment:
Spurious newlines :)

Reply:
we use the new lines to separate blocks of imports. i like them :)

Reply:
Those new lines were fully intended :-) I like to split all imports into blocks so that `github.com/tendermint/tendermint` and `github.com/tendermint/abci` would be in their own block.